packages/plugins/minos-discovery-kong/minos/plugins/kong/kong_client.py

```diff
@@ -0,0 +1,137 @@
+from uuid import (
```

Comment:
Instead of naming this file as `kong_client.py` I think you can simply name it as `client.py`. To avoid a collision with the one that contains de `KongDiscoveryClient` class, you can rename it as `discovery.py`

Reply:
Done!